spec/features/post_show_spec.rb

```diff
@@ -32,7 +32,7 @@
   end
 
   it 'see the post body' do
-    expect(page).to have_content('first comment')
+    expect(page).to have_content('Home')
```

Comment:
- [ ] Great job on writing the missing test cases however as suggested by the previous reviewer please write a test case to see the body of the post i.e. post's `text`.